driver/modern_bpf/helpers/store/auxmap_store_params.h

```diff
@@ -1556,6 +1558,13 @@ static __always_inline void apply_dynamic_snaplen(struct pt_regs *regs,
 	 */
 	unsigned long args[5] = {0};
 	struct sockaddr *sockaddr = NULL;
+	typedef union {
```

Comment:
Multiple similar improvements: don't waste lots of stack space while we just need one of these.